### PR TITLE
Command Updates (detect -> build) + helper text

### PR DESCRIPTION
**Pull request by furqanrydhan**
Migrated detect command to build, deprecated detect (will leave it in the CLI for the next month). Also updated the helper text for multiple commands to see if they feel better

### CODE REVIEW

File: packages/cli/src/cli/index.ts [hunk 1]
```diff
@@ -145,8 +141,8 @@ $$$$$$\\   $$$$$$$\\  $$\\  $$$$$$\\   $$$$$$$ |$$\\  $$\\  $$\\  $$$$$$\\  $$$$
 
   program
     .command("upload")
-    .description("Upload any file or directory using decentralized storage")
-    .argument("[upload-path]", "path to file or directory to upload")
+    .description("Upload any file or directory to decentralized storage (IPFS)")
```
Comment (adam-maj) [inline packages/cli/src/cli/index.ts:144]: This won't always be just IPFS - but this is fine for now

File: packages/cli/src/cli/index.ts [hunk 2]
```diff
@@ -73,9 +73,7 @@ $$$$$$\\   $$$$$$$\\  $$\\  $$$$$$\\   $$$$$$$ |$$\\  $$\\  $$\\  $$$$$$\\  $$$$
 
   program
     .command("build")
-    .description(
-      "Compile contracts and detect implemented thirdweb contract extensions",
-    )
+    .description("Compile contract and detect thirdweb contract extensions")
```
Comment (adam-maj) [inline packages/cli/src/cli/index.ts:76]: `contract` -> `contracts` (this can be used on many contracts at once).

Comment (adam-maj) [reply packages/cli/src/cli/index.ts:76]: `build` also still feels like the wrong command here, but `detect` definitely wasn't right either, so this is fine for now. `info`, `compile` are other ideas

Comment (furqanrydhan) [reply packages/cli/src/cli/index.ts:76]: Why does build feel wrong? info / detect don't make sense. I can see compile as the function but feels like in the context of an app build is more in line with what people think about

**Pull request merged**